modular_skyrat/modules/customization/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -33,6 +33,8 @@
 				applied_style = STYLE_VOX
 			else if((DIGITIGRADE in dna.species.species_traits) && (w_uniform.mutant_variants & STYLE_DIGITIGRADE))
 				applied_style = STYLE_DIGITIGRADE
+			else if(dna.species.id == "teshari")
```

Comment:
`isteshari()`

I'm not going to repeat it anymore, but you should understand how it goes by now.